src/MessageHandler/DeliveryCreatedHandler.php

```diff
@@ -63,17 +63,122 @@ public function __invoke(DeliveryCreated $message)
             return;
         }
 
-        $date = Carbon::instance($delivery->getPickup()->getAfter())
+        $tasks = $delivery->getTasks();
+        $order = $delivery->getOrder();
+        $pickup = $delivery->getPickup();
+        $dropoff = $delivery->getDropoff();
+
+        $puafdt = $pickup->getAfter()->format('H:i');
+        $pubfdt = $pickup->getBefore()->format('H:i');
+        $doafdt = $dropoff->getAfter()->format('H:i');
+        $dobfdt = $dropoff->getBefore()->format('H:i');
+        $date = $pickup->getAfter()->format('Y-m-d H:i');
+        $dateLocal = Carbon::instance($pickup->getAfter())
             ->locale($this->locale)
             ->calendar();
 
-        $message = $this->translator->trans('notifications.delivery_created', ['%date%' => strtolower($date)]);
+        $ownerIsPickupAddr = $delivery->getOwner()->getAddress()->getStreetAddress() === $pickup->getAddress()->getStreetAddress();
+        $title = $delivery->getOwner()->getName();
+        $body = $this->translator->trans('notifications.tap_to_open');
```

Comment:
All this logic of rendering the body should be moved to a Twig template.

Reply:
Too much parsing complexity for a template, I ended up by moving the logic to the message class itself at:
`DeliveryCreated->parseTitleAndBodyForPushNotification(Delivery $delivery, TranslatorInterface $translator): array`

We can try to move to a template from there in the future.